shapeshift/web
# feat: prioritize banxa over onramper

## Description

Makes sure Banxa is always prioritized over Onramper, should both
options be available

## Pull Request Type


## Issue (if applicable)



closes #4914

## Risk



None

## Testing



- Go to fiat ramps -> buy ETH
- Ensure Banxa is displayed before OnRamper

### Engineering



- ^

### Operations



- ^

## Screenshots (if applicable)

<img width="453" alt="image" src="https://github.com/shapeshift/web/assets/17035424/3659ccc3-bb41-4180-95ac-94591a4d7745">
